tests/fast-integration/kyma-environment-broker/client.js

```diff
@@ -153,6 +157,17 @@ class KEBClient {
       return new Error(`error while deprovisioning SKR: ${err.toString()}`);
     }
   }
+
+  getRegion() {
+    let reg = this.region
+    if (!reg) {
+      return ""
+    }
+    if (reg && reg != "") {
+      reg += "/"
+    }
+    return reg
+  }
```

Comment:
I think this is more readable:
```suggestion
  getRegion() {
    if (this.region && this.region != "") {
      return `${this.region}/`;
    }
    return "";
  }
```